pkg/operator/bootstrap.go

```diff
@@ -112,7 +112,7 @@ func RenderBootstrap(
 	spec.OSImageURL = imgs.MachineOSContent
 	spec.Images = map[string]string{
 		templatectrl.EtcdImageKey:            imgs.Etcd,
-		templatectrl.SetupEtcdEnvKey:         imgs.SetupEtcdEnv,
+		templatectrl.SetupEtcdEnvKey:         imgs.MachineConfigOperator,
```

Comment:
missed https://github.com/openshift/machine-config-operator/pull/895/files#r298159078 ? (but I saw you commented in there)

Reply:
yep got it! trying again :)